vendor/github.com/heketi/heketi/client/api/go-client/client.go

```diff
@@ -14,13 +14,15 @@ package client
 
 import (
 	"bytes"
+	"context"
```

Comment:
We need to update vendor with the upstream changes instead of changing vendor directly. cc @misterikkit

Reply:
Thanks Michelle & Jonathon.

I'm not familiar with how to update vendor/, any pointers would be appreciated (here or offline).